### ISSUE TITLE
Builds with Bitcode but Error when archiving 

### ISSUE THREAD

**Issue by pruthvikar** [getditto/rust-bitcode#4]
Hello,

I've tried a few different configurations to see if I could archive successfully, but haven't been able to.

With the default configuration in config.sh, I can build with `ENABLE_BITCODE=Yes`.

However, when I try to archive, I get the following error.

```
ld: could not reparse object file in bitcode bundle: 'Unknown attribute kind (60) (Producer: 'LLVM9.0.0svn' Reader: 'LLVM APPLE_1_1100.0.33.17_0')', using libLTO version 'LLVM version 11.0.0, (clang-1100.0.33.17)' for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

config.sh
```
# 1. Select the best branch from https://github.com/apple/swift-llvm

# $ xcrun -sdk iphoneos swiftc --version
# Apple Swift version 5.1 (swiftlang-1100.0.270.13 clang-1100.0.33.7)
# Target: x86_64-apple-darwin19.0.0

# There is a branch called "stable" which looks promising
# At this time (running Xcode 11.3) it's a couple of months newer than the 5.1 branch
SWIFT_BRANCH="stable"

# 2. Pick/install a working Rust nightly (ideally one where RLS and clippy built)
# 3. Note its date
RUST_NIGHTLY="2019-12-16"

# 4. Get its commit - this is what we will check out to build the iOS version
# rustc --version | cut -d '(' -f2 | cut -d ' ' -f1
RUST_COMMIT="a605441e0"
```

xcrun output
```
➜ xcrun -sdk iphoneos swiftc --version
Apple Swift version 5.1.3 (swiftlang-1100.0.282.1 clang-1100.0.33.15)
Target: x86_64-apple-darwin19.3.0
```

This is with Xcode `Version 11.3.1 (11C504)` which could be the cause.

Let me know if there is any other information you need or configurations you'd like me to try.

**Comment by thombles** [comment]
Hi, I've been able to confirm this using the Xcode I have installed (Version 11.3 (11C29)). The version numbers are slightly different but it's the same error.

```
ld: could not reparse object file in bitcode bundle: 'Unknown attribute kind (60) (Producer: 'LLVM9.0.0svn' Reader: 'LLVM APPLE_1_1100.0.33.16_0')', using libLTO version 'LLVM version 11.0.0, (clang-1100.0.33.16)' for architecture arm64
```

Archiving was working before - I'm not sure which component has changed but clearly we've gotten too far out of sync. I'm about to test your changes in #5 (thank you for that!) and I will see if I can find a config that makes this work again.

**Comment by pruthvikar** [comment]
From reading the [branching scheme](https://github.com/apple/llvm-project/blob/apple/master/apple-docs/AppleBranchingScheme.md). It seems like the paragraph at the bottom suggests that the `apple/stable/*` branches are the ones we should seek to compile against.

> apple/stable/20190104 was generated from the swift-5.1-branch branches in swift-clang, swift-llvm, swift-compiler-rt, swift-clang-tools-extra, and swift-libcxx, with the notable exclusion of swift-lldb,

There are also releases for each version of Swift. The [5.1.3 release](https://github.com/apple/llvm-project/releases/tag/swift-5.1.3-RELEASE) seems like the version we should use as that is the Swift version used in Xcode 11.3.1. I wasn't able to find a nightly that would compile against this, but I would assume that this should produce bitcode compatible binaries.

**Comment by thombles** [comment]
> I wasn't able to find a nightly that would compile against this

I’m seeing the same in my test builds so far. Still experimenting though to work through some build errors. I haven’t made sense of the numbering scheme yet - I can’t tell if Xcode is behind or ahead of Rust’s LLVM.

**Comment by thombles** [comment]
Fixed in #5
